kratos/solving_strategies/schemes/residual_based_pseudo_static_displacement_scheme.h

```diff
@@ -89,11 +89,13 @@ class ResidualBasedPseudoStaticDisplacementScheme
      * @param ThisParameters Parameters with the Rayleigh variable
      */
     explicit ResidualBasedPseudoStaticDisplacementScheme(Parameters ThisParameters)
-      :DerivedBaseType(0.0)
+      :DerivedBaseType(0.0),
+       mRayleighBeta(NODAL_MAUX)
```

Comment:
what does this have to do with this PR?

Reply:
Well, this PR is to make easier to review #3185

Reply:
BEcause the RAYLEIGHT_BETA is not defined in the core, and it gives an error when running cpp tests

Reply:
why did you add it then in the first place?

I am confused ...

Reply:
To be able to create tests in #3185